docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+# Testnet Hard Fork Voting Guide
+
+This guide was last updated for v0.8.2 
+
+As of the v0.8.0 release of Feb. 13, 2017, the hard-fork voting mechanism was implemented for testing on the Decred Testnet. The test vote is estimated to start on Feb 25, 2017 around 1PM CST and last for 7 days. If you wish to participate, guides for Paymetheus and the Command-Line applications can be found here. 
+
+---
+
+## Introduction
+
+There is a two-step process for voting to implement consensus changes that would create a hard fork. Note: the following block intervals are for the testnet, they will differ for mainnet.
```

Comment:
two-phase process instead of of two-step.  We are generally trying to avoid using steps, since there are multiple steps in each phase.